agent/config/config.go

```diff
@@ -247,6 +251,16 @@ type Policy struct {
 	DefaultEvaluationIntervalHCL string `hcl:"default_evaluation_interval,optional" json:"-"`
 }
 
+// PolicyWorkers holds the configuration for the number of policy evaluation
+// workers for each policy type.
+type PolicyWorkers struct {
+	ClusterPtr *int `hcl:"cluster,optional" json:"-"`
+	Cluster    int
+
+	HorizontalPtr *int `hcl:"horizontal,optional" json:"-"`
+	Horizontal    int
+}
```

Comment:
Adding as a general comment rather than something I heavily believe in; but we do have the `config.Policy` struct and maybe these could go within it, rather than having a separate configuration block?

Reply:
I had it there initially, but it looked kind of off:

```hcl
policy {
  dir                = "/opt/nomad-autoscaler/policies"
  default_cooldown   = "2m"
  cluster_workers    = 8
  horizontal_workers = 4
}
```
or nested:
```hcl
policy {
  dir              = "/opt/nomad-autoscaler/policies"
  default_cooldown = "2m"
  workers {
    cluster    = 8
    horizontal = 4
  }
}
```

To me it doesn't seem like they are related, but I can change it if it's better.